src/components/Rating.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+function Rating (props) {
+    const {children} = props;
+
+    function star(children) {
+
+    if (children<0.9) return '☆☆☆☆☆';
```

Comment:
I know that the instructions of this iteration were unclear, but the idea was that if a number is equal or higher than 0.5/1.5/2.5etc..., it should be rounded up, and if it's lower, it should be rounded down.

One way to do that would be creating a variable `rounded` and use the `Math.round()` method, like:
`const rounded = Math.round(children)`

After that, you could've used the `repeat()` method with your variable
`{'★'.repeat(rounded) + "☆".repeat(5 - rounded)}; `

The `repeat()` method constructs and returns a new string which contains the specified number of copies of the string on which it was called, concatenated together.